stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -101,6 +102,16 @@ class Value():
     def __init__(self, typecode_or_type: str, *args: Any, lock: bool = ...) -> None: ...
 
 # ----- multiprocessing function stubs -----
+def active_children() -> List[Process]: ...
+def allow_connection_pickling() -> None: ...
 def cpu_count() -> int: ...
 def freeze_support() -> None: ...
+def get_all_start_methods() -> List[str]: ...
+def get_context(method: Optional[str] = ...) -> BaseContext: ...
+def get_logger() -> Logger: ...
+def get_start_method(allow_none: Optional[bool] = ...) -> str: ...
+def log_to_stderr(level: Optional[str] = ...) -> Logger: ...
 def Manager() -> SyncManager: ...
+def set_executable(executable: str) -> None: ...
+def set_forkserver_preload(module_names: List[str]) -> None: ...
+def set_start_method(method: Optional[str] = ...) -> None: ...
```

Comment:
Only after 3.4, and it doesn't look like the method is Optional.

There is also an undocumented `force: bool = ...` argument.